sherpa/models/regrid.py

```diff
@@ -460,22 +371,18 @@ def overlaps(self, other):
 
     @property
     def grid(self):
-        """
-        Return the grid representation of this dataset. The grid is always a tuple, even if the
-        dataset is 1-D and not integrated. This is due to the existing architecture of Sherpa's
-        model classes and the fact that there is no signature difference among 1-D and 2-D models,
-        as 1-D models can receive 1 or 2 arrays and 2-D models can receive 2 or 4 arrays.
+        """The grid representation of the space.
 
-        The x and y arrays in the grid are one-dimentional representations of the meshgrid obtained
-        from the x and y axis arrays, as in `numpy.meshgrid(x, y)[0].ravel()`
+        The x and y arrays in the grid are one-dimentional
```

Comment:
one-dimentional -> one-dimensional

Reply:
At last: a typo that was not mine but in the original text.